src/schedule.cpp

```diff
@@ -510,6 +522,9 @@ void schedule::apply(module& p) const
     if(enabled(MIGRAPHX_TRACE_COMPILE{}) or enabled(MIGRAPHX_TRACE_SCHEDULE{}))
     {
         p.annotate(std::cout, [&](auto ins) {
+            if(ins->name() == "@param" and not contains(si.weights, ins))
```

Comment:
Condecov says there is no coverage for this line. Is this a false positive?

Reply:
To cover this line, we need one of the env vars (MIGRAPHX_TRACE_COMPILE and MIGRAPHX_TRACE_SCHEDULE) to be set. Do we have a way to set them?